### PR TITLE
darwin: workaround for native accessors

### LINKED ISSUE #136: Native Darwin Support
Due to a bug in cgo we commented out https://github.com/go-gl/glfw/blob/master/v3.1/glfw/native_darwin.go (See Issue #82)

Unfortunately I don't have a mac, can we test if this is still an issue?

### PR DESCRIPTION
**Pull request by james4k**
Resolves #136.

### CODE REVIEW

File: v3.1/glfw/native_darwin.go [hunk 1]
```diff
@@ -1,22 +1,29 @@
...
-//#define GLFW_EXPOSE_NATIVE_COCOA
-//#define GLFW_EXPOSE_NATIVE_NSGL
-//#include "glfw/include/GLFW/glfw3.h"
-//#include "glfw/include/GLFW/glfw3native.h"
-import "C"
+/*
+#define GLFW_EXPOSE_NATIVE_COCOA
+#define GLFW_EXPOSE_NATIVE_NSGL
+#include "glfw/include/GLFW/glfw3.h"
+#include "glfw/include/GLFW/glfw3native.h"
```
Comment (dmitshur) [inline v3.1/glfw/native_darwin.go:7]: Off-topic, but why is GitHub [showing so much has changed](https://cloud.githubusercontent.com/assets/1924134/9560363/024e0304-4dc4-11e5-9a35-7e2989662d68.png) in this diff when in fact it hasn't:

image

File: v3.1/glfw/native_darwin.go [hunk 2]
```diff
@@ -1,22 +1,29 @@
...
 
 // See: https://github.com/go-gl/glfw3/issues/82
```
Comment (dmitshur) [inline v3.1/glfw/native_darwin.go]: This line should be updated to point to https://github.com/go-gl/glfw/issues/136, or describe the issue.

The funcs below are named "workaround", so it'd be great to mention the context for why a workaround is needed.

**Comment by dmitshur** [comment]
One comment above, otherwise looks good. I'll test it locally, and then it'll have a LGTM to merge from me. Thanks!

**Comment by dmitshur** [comment]
LGTM. Tested it locally, it worked well (I got reasonable values; I didn't try to make use of them). Not seeing any issues.

@james4k, I've edited the PR description to say "Resolves #136." so that the issue is closed when this is merged.

**Comment by tapir** [comment]
Great. LGTM.

**Pull request merged**